clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml

```diff
@@ -2405,6 +2405,19 @@
       Out: ""
       RuleId: "remove_target_link_options_cuda_arg"
 
+#'-static-libstdc++' is not supported together with '-fsycl' when using icpx compiler.
+- Rule: rule_libstdc++_target_link_options
+  Kind: CMakeRule
+  Priority: Fallback
+  CmakeSyntax: libstdc++_target_link_options
+  In: target_link_options${empty}(${target} ${libs})
```

Comment:
the rule doesn't distinguish whether gcc/g++/icpx -fsycl is used to build.  So please emit warrning as only -fsycl doen't support -static-libstd++

Reply:
Will be implemented in a separate pr.